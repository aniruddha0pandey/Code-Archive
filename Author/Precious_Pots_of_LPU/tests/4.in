8
38 3
80 49 77 66 30 42 54 23 69 72 98 8 90 26 5 37 78 24 38 87 60 74 97 82 15 6 48 51 20 36 7 61 70 39 10 65 3 35 
37 74 61 
38 4
64 88 1 87 46 25 24 10 100 83 12 29 2 17 31 41 26 79 27 19 73 51 34 96 15 40 22 70 81 21 98 93 72 14 45 60 52 18 
87 31 22 98 
39 2
25 35 97 82 15 76 9 7 84 49 1 53 17 23 42 45 59 34 38 16 89 22 85 21 78 4 50 54 98 18 58 87 13 14 99 40 26 33 56 
34 40 
36 3
90 12 19 88 34 98 58 64 67 11 6 47 9 97 13 2 74 53 41 40 18 99 30 29 60 83 1 25 79 61 93 66 69 38 15 48 
67 11 25 
41 2
44 2 60 99 83 97 98 90 4 5 11 54 63 52 78 58 88 40 81 55 80 70 17 86 35 25 61 85 77 53 56 48 26 23 13 29 64 96 75 42 57 
80 77 
37 3
1 4 66 52 54 56 9 83 53 99 12 16 65 96 48 11 36 91 67 45 3 38 35 39 27 50 40 71 20 31 64 72 46 93 90 87 14 
36 40 14 
39 5
69 39 79 83 28 37 52 64 12 62 89 24 36 81 68 15 78 57 48 85 6 19 61 60 14 88 63 98 26 9 45 74 59 86 8 25 22 18 66 
79 52 81 86 66 
39 2
84 75 50 2 87 14 4 16 66 98 64 22 62 68 29 40 43 86 18 65 67 56 45 15 61 38 24 6 70 79 73 99 88 5 52 59 81 37 32 
40 73 
