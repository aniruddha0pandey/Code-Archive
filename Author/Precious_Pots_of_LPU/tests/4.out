80 49 77 66 30 42 54 23 69 72 98 8 90 26 5 78 24 38 87 60 97 82 15 6 48 51 20 36 7 70 39 10 65 3 35 37 74 61 
64 88 1 46 25 24 10 100 83 12 29 2 17 41 26 79 27 19 73 51 34 96 15 40 70 81 21 93 72 14 45 60 52 18 87 31 22 98 
25 35 97 82 15 76 9 7 84 49 1 53 17 23 42 45 59 38 16 89 22 85 21 78 4 50 54 98 18 58 87 13 14 99 26 33 56 34 40 
90 12 19 88 34 98 58 64 6 47 9 97 13 2 74 53 41 40 18 99 30 29 60 83 1 79 61 93 66 69 38 15 48 67 11 25 
44 2 60 99 83 97 98 90 4 5 11 54 63 52 78 58 88 40 81 55 70 17 86 35 25 61 85 53 56 48 26 23 13 29 64 96 75 42 57 80 77 
1 4 66 52 54 56 9 83 53 99 12 16 65 96 48 11 91 67 45 3 38 35 39 27 50 71 20 31 64 72 46 93 90 87 36 40 14 
69 39 83 28 37 64 12 62 89 24 36 68 15 78 57 48 85 6 19 61 60 14 88 63 98 26 9 45 74 59 8 25 22 18 79 52 81 86 66 
84 75 50 2 87 14 4 16 66 98 64 22 62 68 29 43 86 18 65 67 56 45 15 61 38 24 6 70 79 99 88 5 52 59 81 37 32 40 73 
